site/_posts/2018-07-12-Announcing-Gaprindashvili-4.html.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Announcing Gaprindashvili-4
+author: jprause
+date: 2018-07-12 08:35:39 UTC
+comments: true
+published: true
+tags: releases announcements
+---
+
+We've just built Gaprindashvili-4. This release contains bug fixes, UI tweaks, and stabilization.
+Here are just a few of the things added since Gaprindashvili-3 release:
+
+- ADDED
+  * Introduce support for multi-tab orchestration dialogs [(#17342)](https://github.com/ManageIQ/manageiq/pull/17342)
+  * Support for hidden columns in reports and views [(#17133)](https://github.com/ManageIQ/manageiq/pull/17133)
+  * v2v plugin [(#17529)](https://github.com/ManageIQ/manageiq/pull/17529)
```

Comment:
should we mention tech preview only, or does it not matter for upstream??

Reply:
Thanks @simaishi 
@Fryguy could you give me a more expansive description instead of just "v2v plugin"

Reply:
For upstream we don't have to mention tech preview.

However it's a good idea to expand on the description, e.g. "Add V2V plugin for migration of VMs between different providers"